src/Polly/Polly.csproj

```diff
@@ -33,6 +33,7 @@
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
   <ItemGroup>
+    <Using Remove="System.Net.Http" />
```

Comment:
Did you mean to add this one?

Reply:
yeah thats a known issue with implicit usings. one of the default includes when u do `ImplicitUsings>enable` but it doesnt exist in older frameworks. u can see how common it is https://sourcegraph.com/search?q=context:global+%3CUsing+Remove%3D%22System.Net.Http%22+/%3E&patternType=standard&sm=1